pkg/reconcile/pipeline/api.go

```diff
@@ -0,0 +1,175 @@
+package pipeline
+
+import (
+	"fmt"
+	olmalpha1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
```

Comment:
Let's keep this consistent as olmv1alpha1 https://github.com/redhat-developer/service-binding-operator/pull/919/files#diff-031de41097d908cf1f0f4728e2564fa6f8bc797433c9718a2bef50507012fdcfR6

At some places we have it as olmalpha1